kernel/driver.cc

```diff
@@ -268,8 +268,8 @@ int main(int argc, char **argv)
 		printf("    -s scriptfile\n");
 		printf("        execute the commands in the script file\n");
 		printf("\n");
-		printf("    -c tcl_scriptfile\n");
-		printf("        execute the commands in the tcl script file (see 'help tcl' for details)\n");
+		printf("    -c script\n");
+		printf("        execute the commands in the tcl script file (see 'help script' for details)\n");
```

Comment:
Actually, this is just completely incorrect. The Yosys `script` command does not use Tcl for scripting.

Reply:
At least now I feel like my confusion was legitimate, haha!

When I ran 'yosys -h', the parameter help was the one there that came out of 'driver.cc'. But when I ran 'yosys -h tcl_scriptfile' like it suggested, that wasn't even a command. The only thing I saw when running 'help' directly, was 'script', which definitely was what I wanted.

Have I just massively got my wires crossed here?

Reply:
Where does it suggest to run `yosys -h tcl_scriptfile`? I can't find it.

Reply:
Okay, let me list my numerous mistakes:

I was under the impression that the tcl scripting *was* yosys scripting. I see now that they are two separate things. That clears up a lot of my misconceptions.

Second: my apologies, it was actually suggesting to run 'yosys -h tcl'. I got "no command or cell type tcl" when I ran that.

I've now got to the bottom of *that*: I'm using a nightly build from open tool forge, that is apparently built *without* tcl support, which explains why I never could find the command -- I simply didn't have it 😬

In short: I'm sorry to have wasted your time with my misunderstanding(s)! Maybe all I really should have submitted as a PR is perhaps a conditional compile out for the -c help line if tcl support isn't enabled.

Reply:
> Maybe all I really should have submitted as a PR is perhaps a conditional compile out for the -c help line if tcl support isn't enabled.

Sounds good to me!